src/schema/v2/me/myBids.ts

```diff
@@ -379,3 +396,33 @@ function withSaleInfo(sale): SaleSortingInfo {
     liveBiddingStarted,
   }
 }
+
+function formatGravityLotStandingsToMatchCausalityLotStandings(
+  gravityLotStandings
+): LotStandingResponse {
+  return gravityLotStandings.map((lotStanding) => {
+    const { sale_artwork, id, bidder, highest_bid_amount_cents } = lotStanding
+    return {
+      lot: {
+        bidCount: sale_artwork.bidder_positions_count,
+        reserveStatus: sale_artwork.reserve_status,
+        internalID: id,
+        saleId: bidder.sale.id,
+        soldStatus: getSoldStatus(lotStanding),
+        sellingPriceCents: highest_bid_amount_cents,
+      },
+    }
+  })
+}
+
+function getSoldStatus(lotStanding): string {
+  const { sale_artwork } = lotStanding
+  const { artwork } = sale_artwork
+  if (artwork.forSale) {
```

Comment:
This is probably `artwork` from the JSON response for a lot standing, and not a GraphQL type, so would be using underscored fields from the Gravity API (and not any GraphQL fields).